restore-cognito.md

```diff
@@ -0,0 +1,32 @@
+Prerequisites 
+For creating and executing backups, you can use the npm package called `cognito-backup-restore` 
+(do `npm install cognito-backup-restore` to get the package)
+
+Required IAM permissions
+If you want to create a backup of the users you will need the permission - `ListUsers`
+Restoring the users requires - `AdminCreateUser`
+There might be more roles required to do all the operations for getting and restoring users. If you need a permission added
+speak with Iva or Pol, who will add it through IAM.
+
+**Downloading user data**
+Here do either 1 or 2 depending on the situation.
+
+1. In case there is no cognito user pool or the user pool is empty, we have weekly user backup which executes every sunday at 1am. 
+To find the latest users backup file you need to go to biomage-backups-production S3 bucket and find the latest object in the bucket. 
+There you can download the `data.json` file, which later will be used to import the users to the new/existing user pool.
+
+2. In case the users are still existing in a pool you can create a new backup file from the user pool, which can then be used to import users.
```

Comment:
What about a case 3 where part of the data is corrupted? Have you considered the case when the current Cognito contents are not valid for some reason, but they contain more users than the latest backup, so just restoring the latest Cognito backup wouldn't work, because we would loose users?

Reply:
so if we have users we can just get them as specified in step 2? I don't understand what's the use case here.

Reply:
would importing users to Cognito pool overwrite the current contents of the Cognito pool? Or would it add new users without deleting the users that are already in there?

Reply:
It will add new users, preserving the old ones. Trying to add a user that already exists will ignore the operation and keep the old one.

Reply:
sounds good